internal/provider/resources/user_test.go

```diff
@@ -36,8 +38,8 @@ removed {
 //  1. Provide a value for `api_key` from an API key generated as a user,
 //     not a service account.
 //  2. Provide the user ID as an environment variable: `ACC_TEST_USER_RESOURCE_ID`
-func SkipIfUserResource() (bool, error) {
-	if os.Getenv("ACC_TEST_USER_RESOURCE") == "yes" {
+func SkipUnlessUserResource() (bool, error) {
+	if _, set := os.LookupEnv(envUserResourceID); set {
```

Comment:
Simplifies the logic here a bit, just checking if the user resource ID is set at all. This way we don't need yet another env var set to "yes" to run the tests.